tests/unit/api/bucketPutPolicy.js

```
@@ -78,7 +78,7 @@ describe('putBucketPolicy API', () => {
         });
     });
 
-    it('should return error if policy contains conditions', done => {
+    it.skip('should return error if policy contains conditions', done => {
```

why are we skipping this test?

---

It's not needed: we now handle policy conditions, so we should not send back error any more. I want confirmation to delete it, all good?

---

We should delete if not needed or re-purpose :)

---

Is there a test for checking error is not returned when policy contains conditions?